content-cache-backends-config/docs/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 2024-12-16
+
+### **Added**
+
+- Support for healthcheck-path and healthcheck-interval.
```

Comment:
Maybe mention these are juju configurations.

Reply:
Fixed in 2cb65e0b15a811fd42575839098953e78e2d1296